_alerts/2017-07-12-disruptive-forks.md

```diff
@@ -0,0 +1,86 @@
+bannerclass: "warning"
+---
+*Last updated: 2017-07-12 12:00 UTC.  This page will be updated when new
+information becomes available.  See the [list of updates][].*
+
+On 1 August 2017, Bitcoin confirmation scores may become unreliable for
+an unknown length of time.  This means that any bitcoins you receive
+after August 1st may disappear from your wallet at a later time or be a
+type of bitcoin that other people will not accept as payment.
+
+Once the situation is resolved, confirmation scores will either
+automatically return to their normal reliability or there will be two
+(or more) competing versions of Bitcoin.  In the former case, you may
+return to using Bitcoin normally; in the later case, you will need to
+take extra steps in order to begin safely receiving bitcoins again.
+
+This post currently describes what actions you can take to prepare for
+this situation.  Subsequent to August 1st, we will update this post as
+best we can with relevant information, but you are also advised to
+monitor other Bitcoin [news sites][] and [community resources][] for
+updates and to cross-check all information, as someone may attempt to
+spread false news in order to exploit the situation.
+
+Remember that you alone are responsible for the safety of your bitcoins,
+and that if you lose control of them for any reason, there is nothing
+the operators or contributors to this website can do to help you.
+
+## Before August 1st
+
+1. If you accept bitcoins as payments, we recommend that you stop
+accepting Bitcoin payments at least 12 hours before 00:00 UTC on August
+1st (24 to 48 hours may be safer).  This will give time for all pending
+payments to confirm on the Bitcoin block chain before the event.
+
+1. If you send bitcoins as payments, note that many services may stop
+accepting bitcoins at 00:00 UTC on August 1st or earlier.
+
+1. Be wary of storing your bitcoins on an exchange or other third-party
+wallet.  If they accept transactions during the event, they could lose
+money and will likely spread those loses across all their users.  If
+there end up being two or more competing versions of Bitcoin, then they
+may refuse to give you your bitcoins on versions they don't like.
+
+## On August 1st
+
+1. Do not trust any payments you receive after August 1st until the situation
+is resolved.  No matter how many confirmations the new payment says it
+has, it can disappear from your wallet at any point up until the
+situation is resolved.
+
+1. Try not to send any payments.  Although you should not technically be
+able to lose money this way, wallets may become confused and require
+recovery if you attempt this.
+
+1. Be wary of offers to allow you to invest in the outcome of the event
+by "splitting" your coins.  Some of these offers may be scams, and
+software claiming to split your coins can also steal them.
+
+## After August 1st
+
+No information available yet.  Please wait for multiple trustworthy
```

Comment:
Let's revise this to say that we'll update this section with more information after August 1st.